src/components/card/Card.jsx

```diff
@@ -1,82 +1,79 @@
-    if (this.props.card.assignees.map(element => element.id).includes(memberId) === true) {
-      this.props.removeAssigneeInCard(this.props.card.id, memberId)
-    } else {
-      this.props.addAssignee(this.props.card.id, memberId)
-    }
+  getAssignees() {
+    return (
+      <div>
+        { this.props.card.isFetchingAssignees ? (
+          <Loader message="Loading labels..." />
+        ) : (
+          <AssigneesContainer cardId={this.props.id} displayAssignees displaySelect />
+        )}
+      </div>
+    )
   }
 
-  handleChangeLabels(value) {
-    const labelId = value[value.length - 1].slice(this.props.id.length, value.toString().length)
-    if (this.props.card.labels.map(element => element.id).includes(labelId) === true) {
-      this.props.removeLabelInCard(this.props.card.id, labelId)
-    } else {
-      this.props.addLabel(this.props.card.id, labelId)
-    }
+  getLabels() {
+    return (
+      <div>
+        { this.props.card.isFetchingLabels ? (
+          <Loader message="Loading labels..." />
+        ) : (
+          <LabelsContainer cardId={this.props.id} displayLabels displaySelect />
+        )}
+      </div>
+    )
   }
 
   render() {
     return (
       <div>
         <Layout>
-          <Layout>
-            <Content className="content">
-              <EditArea
-                text={this.props.card.desc}
-                save={(newDesc) => { this.props.updateDesc(this.props.id, newDesc) }}
-                hint="Add a description"
-              />
-              <p>Desc : {this.props.card.desc}</p>
-              <p>DueDate : {this.props.card.dueComplete}</p>
-              <p>Pos : {this.props.card.pos}</p>
-              <p>List ID : {this.props.card.listId}</p>
-              <p>ID : {this.props.id}</p>
-              <p>More...</p>
-            </Content>
-            <Sider className="sider">
-              <Select
-                mode="tags"
-                style={{ width: '60%' }}
-                placeholder="Select users..."
-                onChange={this.handleChangeAssignee}
-                defaultValue={this.props.card.assignees.map(assignee => assignee.fullName)}
-                tokenSeparators={[',']}
-              >
-                {this.props.users.map(user => <Option key={this.props.id + user.id}>{user.fullName}</Option>)}
-              </Select>
-              <Select
-                mode="tags"
-                style={{ width: '60%' }}
-                placeholder="Select labels..."
-                onChange={this.handleChangeLabels}
-                defaultValue={this.props.card.labels.map(label => label.name)}
-                tokenSeparators={[',']}
-              >
-                {this.props.labels.map(label => <Option key={this.props.id + label.id}>{label.name}</Option>)}
-              </Select>
-              <Button type="primary" className="siderButton">Button drive</Button>
-            </Sider>
-          </Layout>
+          { this.props.card.isFetchingAssignees
```

Comment:
You don't need to wait fetching here, do it inside each sub components (assignees, labels, etc)